public/js/chat.js

```diff
@@ -22,9 +22,10 @@
   });
 
   sendbtn.addEventListener('click', () => {
-    console.log(`Sending message to ${room_id_of_other_user}`);
-    socket.emit('sendMessage', { "room": room_id_of_other_user, "message": message.value });
-    message.value = ' ';
```

Comment:
why did you remove this?

Reply:
insert that back.

Reply:
I haven't removed it. I have just added it inside an if statement's true part.

Reply:
the deployment dint replicate. thats why I asked.